src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceApiServiceSpec.scala

```diff
@@ -288,397 +185,419 @@ class WorkspaceApiServiceSpec extends BaseServiceSpec with WorkspaceApiService w
   "WorkspaceService Passthrough Tests" - {
 
     "Passthrough tests on the /workspaces path" - {
-      "MethodNotAllowed error is not returned for HTTP GET and POST methods" in {
```

Comment:
POST for this isn't actually a passthrough, so the test has been moved to a new section below.